dmorg.tex

```diff
@@ -150,6 +150,32 @@ \subsubsection{Hardware Configuration Control}
 
 The sizing model for compute hardware purchasing is detailed in \citeds{LDM-144}, \citeds{LDM-141}, and \citeds{LDM-138}.
 
+\subsection {Release Management } \label{sect:release}
+
+
+The DMCCB maintains  the release plan, \citeds{LDM-564},  synchronized with  the project milestones.
```

Comment:
Seems like a lot of unnecessary white space.

What does this “maintains” mean? Do date, I have personally been updating LDM-564 based on the contents of PMCS — will the DM-CCB be taking that over?

Reply:
(Also, despite it's name, LDM-564 describes a great many things that aren't software releases... I'm not sure if that's relevant to the above.)

Reply:
The changes that may be required by LDM-564 should be addressed separately.
At the moment, I see LDM-564 as the release plan for DM, that goes in parallel to the test plan, LDM-503.
If this is not correct, we can propose a different LDM document for the release plan, and refer to it from LDM-294 section 3.6.
In any cases, I should take care of this document as release manager, keeping in mind that release milestones shall come from P6.

Reply:
Well, I think in this case the title and the contents of LDM-564 are at variance. Specifically, only a couple of the subsections of LDM-564 §3 actually refer to releases. I agree with you that we should fix that. I am ok with you writing in LDM-294 what you think the situation _should_ be (which is effectively what you're doing here), providing that you _also_ file a Jira ticket to over the work of making LDM-564 align with your wishes.

Reply:
It seems to me that the content of LDM-564 is in line with the title. However the document need to be updated in order to be coherent with the proposed section in LDM-294. Issue DM-17001 has been opened.

Reply:
The first few items in §3 of LDM-564 are:

- Science Platform with WISE data in PDAC
- HSC reprocessing
- Alert generation validation
- AuxTel DAQ integration functionality test
- Aux Tel DAQ interface Integration Verification and Spectrograph Operations Rehearsal
- Alert distribution validation

All of these involved released versions of software, but none of these describe DM releases.